test/models/rpm_test.rb

```diff
@@ -404,6 +404,12 @@ def test_search_compare_lt_gte
       assert_equal expected, results.map(&:pulp_id).sort
     end
 
+    def test_evr
+      rpm = Rpm.where(nvra: "abc123-2-1.0.0-1.0").first
+
+      assert_equal "(1,\"{\"\"(1,)\"\",\"\"(0,)\"\",\"\"(0,)\"\"}\",\"{\"\"(1,)\"\",\"\"(0,)\"\"}\")", rpm.evr
+    end
```

Comment:
Could you test updating an attribute on a record and getting back an appropriate updated evr string?

Reply:
ideally there is a test for each of the attributes that affects the evr string